compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -948,6 +948,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool opCodeIsNoOpOnThisPlatform(TR::ILOpCode &opCode);
 
+   bool supportsPrefetch() { return getS390ProcessorInfo()->supportsArch(TR_S390ProcessorInfo::TR_z10); }
```

Comment:
I don't see this function used anywhere in this commit or the existing OMR codebase.  Was it added as an anticipated convenience for future commits?  If so, I think the name could use some refinement as "prefetch" is too generic a term (unless on Z it truly isn't, so correct me if I'm wrong).